metaflow/plugins/__init__.py

```diff
@@ -9,8 +27,15 @@ def get_plugin_cli():
     from . import package_cli
     from .aws.batch import batch_cli
 
-    return [package_cli.cli,
-            batch_cli.cli]
+    return ext_plugins.get_plugin_cli() + [package_cli.cli, batch_cli.cli]
+
+
+def _merge_lists(decos, overrides, attr):
```

Comment:
Maybe you can leave a short comment on usage of `merge_lists`. One thing to bring out would be to prefer entities from `metaflow_custom` as overrides - which is implicit from your usage throughout this file. WDYT?

Reply:
Done